app/views/coronavirus/pages/_actions.html.erb

```diff
@@ -1,18 +1,18 @@
 <div class="app-side">
   <div class="app-side__actions">
     <%= form_with(url: publish_coronavirus_page_path(@page.slug)) do %>
-      <%= render "govuk_publishing_components/components/button", { text: t("coronavirus.pages.actions.publish") } %>
+      <%= render "govuk_publishing_components/components/button", { text: t("coronavirus.pages.show.actions.publish") } %>
```

Comment:
I'm a bit confused by this key change. It looks like it has switched to being incorrect where you'd expect a view of `app/views/coronavirus/pages/_actions.html to have keys under `coronavirus.pages.actions`. 

I see this carries on a bit through the file so thought I'd drop this first comment in so we can discuss.

Reply:
I can move the view.